### PR TITLE
My contributions page and update dialog/notification texts

### PR DESCRIPTION
**Pull request by cooltey**
**My contributions page**
https://phabricator.wikimedia.org/T207339

**Dialog/notification concept**
https://phabricator.wikimedia.org/T217369

### CODE REVIEW

**Comment by cooltey** [comment]
Just an FYI for testing purposes, I've added developer preferences for showing notifications and dialogs, and developers can test usability for notification if the app is not open. (you can close the app and then click on the notification)

File: app/src/main/java/org/wikipedia/editactionfeed/MyContributionsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        override fun onBindViewHolder(holder: ItemViewHolder, pos: Int) {
+            holder.bindItem(languageList[pos])
+        }
+    }
+
+    private inner class ItemViewHolder internal constructor(itemView: View) : DefaultViewHolder<View>(itemView) {
+        internal fun bindItem(langCode: String) {
+            ViewUtil.formatLangButton(itemView.languageCode, langCode, LANG_BUTTON_TEXT_SIZE_SMALLER, LANG_BUTTON_TEXT_SIZE_LARGER)
+            itemView.languageCode.text = langCode
+            itemView.languageTitle.text = WikipediaApp.getInstance().language().getAppLanguageLocalizedName(langCode)
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/editactionfeed/MyContributionsFragment.kt]: It needs to use Canonical name : https://app.zeplin.io/project/57a120b91998d8977642a238/screen/5c77dc34d693e3bc7268a88b

File: app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsActivity.kt [hunk 2]
```diff
@@ -31,25 +32,27 @@ class AddTitleDescriptionsActivity : SingleFragmentActivity<AddTitleDescriptions
...
             }
 
             R.id.menu_help -> {
                 FeedbackUtil.showAndroidAppEditingFAQ(baseContext)
                 true
             }
-            R.id.menu_my_contributions -> {
-                // TODO: go to My contributions
-                true
-            }
```
Comment (sharvaniharan) [inline app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsActivity.kt]: Are we sure this must be removed? If I remember right we need to be able to access contributions from task list screen, Add Descriptions overflow and Add Translations overflow..

Comment (cooltey) [reply app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsActivity.kt]: Thanks! Not sure why I removed it, but I add it back now 👍

**Comment by cooltey** [comment]
In fc4390e, I've also updated the `getCanonicalName()` and `getLocalizedName()` methods in `AppLanguageLookUpTable` since the Chinese is using in many countires instead of only in China.

**Pull request merged**